### ISSUE TITLE
Numbers in variable names do not work

### ISSUE THREAD

**Issue by micahhausler** [kopeio/kexpand#5]
```
$ cat values.yaml
s3bucket: my-bucket
$ kexpand expand my-config.yaml -f values.yaml
apiVersion: v1
kind: ConfigMap
metadata:
  name: my.config
data:
  s3.bucket: $((s3bucket))
```

**Comment by denniswebb** [comment]
@micahhausler The above PR will fix your issue if you want to pull the changes yourself.

```
/tmp/issue5 $ cat values.yaml
s3Bucket: my-bucket
/tmp/issue5 $ cat my-config.yaml
apiVersion: v1
kind: ConfigMap
metadata:
  name: my.config
data:
  s3Bucket: $((s3Bucket))
/tmp/issue5 $ kexpand expand -f values.yaml my-config.yaml
apiVersion: v1
kind: ConfigMap
metadata:
  name: my.config
data:
  s3Bucket: my-bucket
```

**Comment by justinsb** [comment]
Merged #6, so fixed!  Thanks @denniswebb
